week3/intro.txt

```diff
@@ -3,4 +3,4 @@ My five favorite things are:
 - Cats
```

Comment:
Hi megan,
Im Aaron and im reviewing your code. I like that cats is part of your favorite things list. I have 2 cats who love to make a mess.